velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2863,57 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+// The test for the alias now() is integrated into this test.
+TEST_F(DateTimeFunctionsTest, currentTimestamp) {
+  const auto evaluateCurrentTimestamp = [&](const std::string& functionName,
+                                            int64_t expectedTimestamp,
+                                            const std::string& timeZone) {
+    auto emptyRowVector = makeRowVector(ROW({}), 1);
+
+    core::TypedExprPtr callExpr = std::make_shared<const core::CallTypedExpr>(
+        TIMESTAMP_WITH_TIME_ZONE(),
+        std::vector<core::TypedExprPtr>{},
+        functionName);
+
+    auto actual = evaluate(callExpr, emptyRowVector);
+
+    auto expected =
+        makeTimestampWithTimeZoneVector(expectedTimestamp, timeZone.c_str());
+
+    ASSERT_EQ(expected->size(), actual->size());
+    ASSERT_TRUE(expected->type()->equivalent(*actual->type()))
+        << "Expected " << expected->type()->toString() << ", but got "
+        << actual->type()->toString();
+    ASSERT_TRUE(actual->compare(expected.get(), 0, 0) < 5)
+        << "timestamp: expected " << expected->toString(0) << ", but got "
+        << actual->toString(0)
+        << ", which is larger than the 5 milliseconds acceptable lag.";
+
+    if (expected->size() == 2) {
```

Comment:
Isn't expected->size() always 2 since we expect Timestamp with timezone always ? It might be better to CHECK that size instead.

Reply:
I mentioned this problem before, I tried to find the cause of the problem but I failed, so I wrote the code like that. When we execute this function first time ` evaluateCurrentTimestamp(
      "current_timestamp", Timestamp::now().toMillis(), "+00:00");`, its `expected->size() == 1`. But after that, the second time and all the following executions of evaluateCurrentTimestamp(), the size is correct, which is `expected->size() == 2`. It is really weird.

Reply:
Ok, I figured out the problem, my vector `actual` and `expected` has only 1 element, but this element is a vector and has two value. e.g. 
```
[          ] [ INFO ]{1687227070582, 0}{1687227070582, 0}
[          ] [ INFO ]{1687227070582, 0}{1687227070582, 0}
[          ] [ INFO ]{1687227070582, 421}{1687227070582, 421}
[          ] [ INFO ]{1687227070582, 421}{1687227070583, 421}
[          ] [ INFO ]{1687227070583, 1320}{1687227070583, 1320}
```
So I only need to compare the first element in `actual` and `expected`...

This PR is ready to review again @aditi-pandit